sdk/template/template-dpg/package.json

```diff
@@ -0,0 +1,138 @@
+{
+  "name": "@azure/template-dpg",
+  "version": "1.0.0-beta.1",
+  "description": "Widget Service",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "react-native": {
+    "./dist/index.js": "./dist-esm/src/index.js"
+  },
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./types/src/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist-esm/src/index.js"
+    },
+    "./api": {
+      "types": "./types/src/api/index.d.ts",
+      "import": "./dist-esm/src/api/index.js"
+    },
+    "./rest": {
+      "types": "./types/src/rest/index.d.ts",
+      "import": "./dist-esm/src/rest/index.js"
+    }
+  },
+  "//metadata": {},
+  "types": "types/latest/template-dpg.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "echo Obsolete",
+    "build:test:browser": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
+    "build:test:node": "tsc -p . && dev-tool run bundle",
+    "build:test": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
+    "build:output": "node scripts/renameOutput.mjs",
+    "build": "dev-tool customization apply && npm run build:test && api-extractor run --local && npm run build:output",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* temp types *.tgz *.log",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"sources/customizations/**/*.ts\" \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "tsp compile --config \"./typespec\" ./typespec/main.tsp",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint README.md package.json api-extractor.json src test --ext .ts,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint README.md package.json api-extractor.json src test --ext .ts,.javascript,.js",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build && npm run integration-test:browser",
+    "test:node": "npm run clean && tsc -p . && npm run integration-test:node",
+    "test": "npm run clean && tsc -p . && npm run unit-test:node && dev-tool run bundle && npm run unit-test:browser && npm run integration-test",
+    "unit-test:browser": "dev-tool run test:browser",
+    "unit-test:browser:debug": "karma start karma.conf.cjs --single-run",
```

Comment:
I think this can be updated similar to https://github.com/Azure/azure-sdk-for-js/blob/64e1a672ceeb3341f33d6130be85a30feac8fb93/sdk/openai/openai/package.json#L56

```suggestion
    "unit-test:browser": "dev-tool run test:browser -- karma.conf.cjs",
```